src/GlobalScreen/Scope/Notification/Factory/StandardNotification.php

```diff
@@ -1,196 +1,91 @@
-    /**
-     * @var string
-     */
-    private $action;
-    /**
-     * @var DateTimeImmutable
-     */
-    protected $date;
-    /**
-     * @var string
-     */
-    protected $summary;
-    /**
-     * @var Closure
-     */
-    protected $close_action_callback;
-
-
-    /**
-     * @inheritDoc
-     */
-    public function withDate(DateTimeImmutable $date_time_immutable) : isItem
-    {
-        $clone = clone $this;
-        $clone->date = $date_time_immutable;
-
-        return $clone;
-    }
-
-
-    /**
-     * @inheritDoc
+     * UI Component mapping to this item
+     *
+     * @var NotificationItem
      */
-    public function getDate() : DateTimeImmutable
-    {
-        return $this->date;
-    }
-
+    private $notification_item;
 
     /**
-     * @inheritDoc
-     */
-    public function withCloseActionCallback(Closure $callback) : isItem
-    {
-        $clone = clone $this;
-        $clone->close_action_callback = $callback;
-
-        return $clone;
-    }
-
-
-    /**
-     * @inheritDoc
-     */
-    public function getCloseActionCallback() : Closure
-    {
-        return $this->close_action_callback;
-    }
-
-
-    /**
-     * @inheritDoc
-     */
-    public function withAction(string $action) : isItem
-    {
-        $clone = clone $this;
-        $clone->action = $action;
-
-        return $clone;
-    }
-
-
-    /**
-     * @inheritDoc
+     * Amount of old notes, the notification contains.
+     * @see hasAmount
+     *
+     * @var int
      */
-    public function hasAction() : bool
-    {
-        return is_string($this->action);
-    }
-
+    private $old_amount = 0;
 
     /**
-     * @inheritDoc
+     * Amount of old notes, the notification contains.
+     * @see hasAmount
+     *
+     * @var int
      */
-    public function getAction() : string
-    {
-        return $this->action;
-    }
-
+    private $new_amount = 1;
 
     /**
-     * @inheritDoc
+     * @param NotificationItem $notification_item
+     * @return StandardNotification
      */
-    public function withSymbol(Symbol $symbol) : canHaveSymbol
+    public function withNotificationItem(NotificationItem $notification_item) : StandardNotification
     {
         $clone = clone $this;
-        $clone->symbol = $symbol;
-
+        $clone->notification_item = $notification_item;
         return $clone;
     }
 
 
     /**
-     * @inheritDoc
+     * @return NotificationItem
      */
-    public function hasSymbol() : bool
+    public function getNotificationItem() : NotificationItem
     {
-        return ($this->symbol instanceof Symbol);
+        return $this->notification_item;
     }
 
-
-    /**
-     * @inheritDoc
-     */
-    public function getSymbol() : Symbol
-    {
-        return $this->symbol;
-    }
-
-
     /**
-     * @inheritDoc
+     * @inheritdoc
      */
-    public function withAdditionalAction(string $title, string $action) : isItem
+    public function withOldAmount(int $amount = 0) : StandardNotification
```

Comment:
'old' is in the sense of 'read' or 'already displayed'?

Reply:
yes. Though long about the naming. withDisplayed, withNotified, withSeen came to my mind. After all, I decided for old. Maybe not the best choice...